### PR TITLE
Resolve technical and editorial issues

### PR DESCRIPTION
**Pull request by ChristinaRookie**

### CODE REVIEW

**Comment by arthurattwell** [comment]
@ChristinaRookie Can you remove the `gulpfile.js`, `package.json` and `package-lock.json` from this commit, please? You've committed old versions back in. (Unless I'm missing something of course.)

File: public-health/text/01-02.md [hunk 1]
```diff
@@ -89,8 +89,14 @@ If the prevalence and incidence are looked at when planning services, one might
...
-DALY
-= Years of life lost due to early death + Years of life lived with disability  
+$$ \begin{equation}
+\text{DALY =} 
+\begin{split}
+&\text{Years of life lost due to early death} \\
+& \qquad\qquad\qquad\qquad+ \\
+& \text{Years of life lived with disability}\\
+\end{split}
+\end{equation} $$
```
Comment (arthurattwell) [inline public-health/text/01-02.md]: @ChristinaRookie Would this look better?

```
$$
\begin{align}
\text{DALY}\quad=\quad
&\text{Years of life lost}\\
&\text{due to early death} \quad +\\
&\text{Years of life lived}\\
&\text{with disability}
\end{align}
$$
```

File: public-health/text/01-02.md [hunk 2]
```diff
@@ -230,10 +236,21 @@ A person lives with bipolar disorder for 20 years and then dies 5 years prematur
...
+& (5) \\
+& + \\ 
+& \text{years of life lived with bipolar disorder} \\
+& (20 × 0.4) \\
+& = \\
+& 5+(20 × 0.4) \\
+& = \\
+& \text{13 DALYs lost}
+\end{align*}
+$$
```
Comment (arthurattwell) [inline public-health/text/01-02.md]: @ChristinaRookie Would this look better?

```
$$
\begin{align}
\text{DALY} \quad = \quad
&\text{years of life lost} (5) \quad +\\
&\text{years of life lived} \\
&\text{with bipolar disorder} \\
& (20 × 0.4) \\
= \quad
& 5+(20 × 0.4)\\
= \quad
& \text{13 DALYs lost}
\end{align}
$$ 
```

File: public-health/text/01-02.md [hunk 3]
```diff
@@ -378,15 +395,31 @@ Often, the denominator is taken from published mid-year population estimates. Th
...
+
+$$ 
+\begin{align*}
+&\text{Mode: 2, 2, 2, 3, 3, 6, 9, 10, 28, 35} \\
+&= \text{2 minutes (there are 3 values of 2)}
+\end{align*}
+$$
+
```
Comment (arthurattwell) [inline public-health/text/01-02.md]: @ChristinaRookie Would this look better?

```
$$ 
\begin{align*}
\text{Mean} & = \frac{(10+2+2+2+3+\\35+3+6+28+9)}{10} \\
&= \text{10.3 minutes}
\\
\\
\text{Median:} & \quad 2, 2, 2, 3, 3, 6, 9, 10, 28, 35 \\
&= \frac{(3+6)}{2} \\
&= \text{4.5 minutes}
\\
\\
\text{Mode:} & \quad 2, 2, 2, 3, 3, 6, 9, 10, 28, 35 \\
&= \text{2 minutes (there are 3 values of 2)}
\end{align*}
$$
``

**Comment by arthurattwell** [comment]
Some notes:

1. The TOC structure is great now. I now need to style the part titles. Rather than trying to target their position in the list tree, I'll add explicit classes in the TOC meta and target these in CSS.
2. For budget reasons we have to live with Part headings being the same level as chapter headings, even though strictly parts should be h1 and chapters h2. (It would just cost more than it's worth to write CSS that allowed for a deeper heading hierarchy for this book.) This is a small concession to accessibility.

    That said, **unit objectives** should not be the same h2 heading level at the unit heading. They should be h3s. I can fix this when I do the styling.

3. Figure 5.5 is too small to be readable. I don't think it shoudl be '.fixed'. This is for @nkoeries to decide about. Maybe there are knock-on issues that make that impossible.

**Comment by nkoeries** [comment]
@arthurattwell 
1. I'm okay with this. 
2. In the other books, the unit objectives are h2 so I went with that. I'm okay with it being h3s. I didn't want it to deviate too much from the standard book structure. 
3. The image is the answer to the question it follows, and that is why it is fixed. What size would make the image readable (so I can go back to the illustrator) or is there another way to resolve it?

**Comment by ChristinaRookie** [comment]
I've pushed those MathJax fixes now, thanks @arthurattwell. It looks much better.

I agree that the 'objectives' should not be H2s, you'll probably want to make case studies H3s as well, as they fall within the units. I can do this if necessary, although we haven't costed for this work.

**Comment by ChristinaRookie** [comment]
@arthurattwell I removed the old files for my repo, I hope it's all okay now.

**Comment by arthurattwell** [comment]
> @arthurattwell I removed the old files for my repo, I hope it's all okay now.

@ChristinaRookie By deleting them, this PR now deletes the files, too. You'll need to get the latest versions of the files from the upstream master branch (e.g. download them from GitHub) and commit them to your fork.

**Comment by arthurattwell** [comment]
>     3. The image is the answer to the question it follows, and that is why it is fixed. What size would make the image readable (so I can go back to the illustrator) or is there another way to resolve it?

I suggest removing `.fixed` and referring to the figure number in the answer. E.g. 'See Figure 5-5 for this epidemiological triangle.'

**Comment by ChristinaRookie** [comment]
> > @arthurattwell I removed the old files for my repo, I hope it's all okay now.
> 
> @ChristinaRookie By deleting them, this PR now deletes the files, too. You'll need to get the latest versions of the files from the upstream master branch (e.g. download them from GitHub) and commit them to your fork.

@arthurattwell I pushed the latest EBW files now.

**Comment by arthurattwell** [comment]
> I agree that the 'objectives' should not be H2s, you'll probably want to make case studies H3s as well, as they fall within the units. I can do this if necessary, although we haven't costed for this work.

@ChristinaRookie Thanks, good point. I'll see if I can do this quickly while styling. I've added this to my list in #59.

**Pull request merged**